testnets/kyvedevnet/assetlist.json

```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "../../assetlist.schema.json",
+    "chain_name": "kyvedevnet",
+    "assets": [
+      {
+        "description": "The native utility token of the Korellia devnet version of KYVE.",
+        "denom_units": [
+          {
+            "denom": "tkyve",
+            "exponent": 0
+          },
+          {
+            "denom": "kyve",
+            "exponent": 9
```

Comment:
that other one was changed to a 6. Is this supposed to be 9?

Reply:
Just confirming but looks like it should change to 6 as well.

Reply:
Korellia/devnet is correct at 9 decimal places. Kaon/testnet is intentionally moving to 6 decimal places